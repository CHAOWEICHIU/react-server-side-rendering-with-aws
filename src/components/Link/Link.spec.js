'use strcit'

import React            from 'react'
import { shallow }      from 'enzyme'
import renderer         from 'react-test-renderer'
import StyledLink       from './'
import { MemoryRouter } from 'react-router-dom'
import 'jest-styled-components'

it('<StyledLink />', ()=>{
  const tree = renderer.create(
    <MemoryRouter >
      <StyledLink to="good">Good</StyledLink>
    </MemoryRouter>
  ).toJSON()
  expect(tree).toMatchStyledComponentsSnapshot()
})
