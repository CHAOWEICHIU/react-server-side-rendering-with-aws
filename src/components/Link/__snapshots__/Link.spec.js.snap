// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledLink /> 1`] = `
.biZewx {
  display: inline-block;
  padding: 10px 30px;
  margin: 5px;
  font-size: 40px;
  color: #696969;
  border: 2px solid rgb(42,184,241);
  font-weight: 100;
  letter-spacing: 4px;
  background-color: transparent;
  border-radius: 200px;
  transition: color 0.3s;
  -webkit-transition: color 0.3s;
  text-decoration: none;
}

.biZewx:hover {
  text-decoration: none;
  color: #a4a4a4;
}

.biZewx:focus {
  text-decoration: none;
  color: white;
}

.biZewx:active {
  text-decoration: none;
  color: green;
}

.biZewx:visited {
  text-decoration: none;
  color: none;
}

<a
  className="biZewx"
  href="good"
  onClick={[Function]}
>
  Good
</a>
`;
